doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_sdfw_sdrfw_update.rst

```diff
@@ -0,0 +1,54 @@
+.. _ug_nrf54h20_suit_sdfw_sdrfw_update:
```

Comment:
The image also needs to be remade

Reply:
What would you like to change in the image?

Reply:
It will be updated in a separate PR